_articles/maintaining-balance-for-open-source-maintainers.md

```diff
@@ -0,0 +1,220 @@
+
+So, what is personal ecology? As <a href="https://rockwoodleadership.org/nonprofit-four-day-workweek-can-take-care-still-change-world/#:~:text=personal%20ecology%3A%20maintaining%20balance%2C%20pacing%20and%20efficiency%20to%20sustain%20your%20energy%20over%20a%20lifetime%20of%20activism">described by the Rockwood Leadership Institute</a>, it involves "<strong>maintaining balance, pacing, and efficiency to sustain our energy over a lifetime</strong>." This framed our conversations, helping maintainers recognize their actions and contributions as parts of a larger ecosystem that evolves over time. Burnout, a syndrome resulting from chronic workplace stress as [defined by the WHO](https://icd.who.int/browse11/l-m/en#/http://id.who.int/icd/entity/129180281), is not uncommon among maintainers. This often leads to a loss of motivation, an inability to focus, and a lack of empathy for the contributors and community you work with.
+
+<aside markdown="1" class="pquote">
+  <img src="https://avatars.githubusercontent.com/gabek?s=180" class="pquote-avatar" alt="avatar">
+  I was unable to focus or start on a task. I had a lack of empathy for users.
+  <p markdown="1" class="pquote-credit">
+— <a href="https://github.com/gabek">@gabek</a>, maintainer of the Owncast live streaming server, on the impact of burnout on his open source work
+  </p>
+</aside>
+
+By embracing the concept of personal ecology, maintainers can proactively avoid burnout, prioritize self-care, and uphold a sense of balance to do their best work.
+
+## Tips for Self-Care and Avoiding Burnout as a Maintainer:
+
+### Identify your motivations for working in open source
+
+Take time to reflect on what parts of open source maintenance energizes you. Understanding your motivations can help you prioritize the work in a way that keeps you engaged and ready for new challenges. Whether it's the positive feedback from users, the joy of collaborating and socializing with the community, or the satisfaction of diving into the code, recognizing your motivations can help guide your focus.
+
+### Reflect on what causes you to get out of balance and stressed out
+
+It's important to understand what causes us to get burned out. Here are a few common themes we saw among open source maintainers:
+
+* **Lack of positive feedback:** Users are far more likely to reach out when they have a complaint. If everything works great, they tend to stay silent. It can be discouraging to see a growing list of issues without the positive feedback showing how your contributions are making a difference.
+
+<aside markdown="1" class="pquote">
+  <img src="https://avatars.githubusercontent.com/thisisnic?s=180" class="pquote-avatar" alt="avatar">
+  I was unable to focus or start on a task. I had a lack of empathy for users.
+  <p markdown="1" class="pquote-credit">
+— <a href="https://github.com/thisisnic">@thisisnic</a>, maintainer of Apache Arrow
+  </p>
+</aside>
```

Comment:
This quote was previously attributed by `@gabek`.

Reply:
Strange, it shows up as @gabek in the rendered page. Do you know what's going on @leereilly ?
<img width="538" alt="Screenshot 2023-08-02 at 9 26 37 AM" src="https://github.com/github/opensource.guide/assets/617994/85fd574c-787b-4827-b366-1e3270e1ed57">

Reply:
Ohhh I see the quote was copied incorrectly below again. It should be 
>“Sometimes it feels a bit like shouting into the void and I find that feedback really energizes me. We have lots of happy but quiet users” 
<img width="532" alt="Screenshot 2023-08-02 at 9 29 07 AM" src="https://github.com/github/opensource.guide/assets/617994/a2afabcd-daf6-4141-8acb-9501b049837a">